reporting/bufferSize.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE HTML>
+<meta charset=utf-8>
+<title>Reporting: Buffer size</title>
+<link rel="author" title="Paul Meyer" href="paulmeyer@chromium.org">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<script>
+  // Test the buffer size (100) of ReportingObserver.
+  async_test(async function(test) {
+    for (i = 0; i != 110; ++i)
+      await test_driver.generate_test_report("" + i);
+
+    var observer = new ReportingObserver(function(reports) {
+      test.step(function() {
+        // Only (the most recent) 100 reports should be observed, even though
+        // 110 were buffered.
```

Comment:
nit: so we're expecting reports 10-110? Can the loop go through that range instead of adding "10" to each expected report? I think that might be a bit clearer.

Reply:
I could, but then I would have to decrement 10 from the index each time instead. I don't think I agree that that would be clearer.

Reply:
Yep, my fault, I missed that detail. Thanks.